.github/ISSUE_TEMPLATE/new-ontology.yml

```diff
@@ -0,0 +1,249 @@
+title: Request for new ontology [NAME]
+labels: [ new ontology ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here: http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html
+  - type: input
+    id: title
+    attributes:
+      label: Title
+      description: What is the full title of your ontology? Please do not include redundant acronyms.
+      placeholder: ex. Chemical Entities of Biological Interest
+    validations:
+      required: true
+  - type: input
+    id: slug
+    attributes:
+      label: Short Description
+      description: Please provide a short description of your ontology in one sentence.
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Please provide a long description of your ontology using english prose. A good description covers the kinds of entities in the ontology, the motivation for creating an ontology, how it relates to other OBO ontologies, and the intended users.
+    validations:
+      required: true
+  - type: input
+    id: idspace
+    attributes:
+      label: Identifier Space
+      description: What is the prefix for the identifier space associated with your ontology? This is typically all capital letters. This *must* not conflict with any entries in the [Bioregistry](https://bioregistry.io/registry) nor [BioPortal](https://bioportal.bioontology.org). See OBO Principle [FP-003 "URI/Identifier Space"](https://obofoundry.org/principles/fp-003-uris.html).
+      placeholder: ex. GO
+    validations:
+      required: true
+  - type: dropdown
+    id: license
+    attributes:
+      label: License
+      description: What license does your ontology have? The OBO Foundry only considers ontologies with open licenses. See OBO Principle [FP-001 "Open"](https://obofoundry.org/principles/fp-001-open.html).
+      options:
+        - CC0
+        - CC-BY 4.0
+        - CC-BY 3.0
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        # Links
+
+        This section of the ontology request form contains links to various resources associated with your ontology.
+  - type: input
+    id: repository
+    attributes:
+      label: Source Code Repository
+      description: What is the URL of the repository? Typically, this will be a link to GitHub.
+      placeholder: ex. https://github.com/obophenotype/uberon/
+    validations:
+      required: true
+  - type: input
+    id: homepage
+    attributes:
+      label: Homepage
+      description: What is the homepage of your ontology? If you don't have a specific homepage, you should consider using GitHub pages to make one! Otherwise, include a duplicate of the repository in this field.
+      placeholder: ex. https://uberon.github.io
```

Comment:
Outdated example, should be updated/redirected, use https://obophenotype.github.io/uberon/

Reply:
Fixed in [706f12a](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1824/commits/706f12a9b1b82c52f3e4eba17dbf76d1cf1822bd)